extensions/Khronos/KHR_lights/README.md

```diff
@@ -0,0 +1,173 @@
+For light types that have a direction (`directional` and `spot` lights), the light's direction is defined as the vector `(0.0, 0.0, -1.0)` and the rotation of the node orients the light accordingly.
+
+`ambient` lights are only valid referenced by a scene.
+
+### Adding Light Instances to Scenes
+
+`ambient` lights have no position and no orientation. These lights are placed in the scene by defining an `extensions` property of a scene, and within that a `KHR_lights` property that identifies a light using its `light` property.
+
+`directional`, `point`, `spot` lights are only valid referenced by a node.
+
+```javascript
+"scenes" : [
+    {
+        "extensions" : {
+            "KHR_lights" : {
+                "light" : 0
+            }
+        }
+    }            
+]
+```
+
+### Light Types
+
+#### Light Shared Properties
+
+Table 1. Light Shared Properties
+
+| Property               | Description                     | Default Value | Applies To                                |
+|:-----------------------|:--------------------------------|:--------------|:------------------------------------------|
+| `color`                | RGB value for light's color.    | [0.0,0.0,0.0] | `ambient`, `directional`, `point`, `spot` |
+| `type`                 | Declares the type of the light. | Mandatory     | `ambient`, `directional`, `point`, `spot` |
+| `constantAttenuation`  | Constant attenuation.           | 1.0           | `point`, `spot`                           |
+| `linearAttenuation`    | Linear attenuation.             | 0.0           | `point`, `spot`                           |
+| `quadraticAttenuation` | Quadratic attenuation.          | 0.0           | `point`, `spot`                           |
+| `falloffAngle`         | Falloff angle in radians.       | PI / 2.0      | `spot`                                    |
+| `falloffExponent`      | Falloff exponent.               | 0.0           | `spot`                                    |
+
+#### Ambient
+
+Ambient lights define constant lighting throughout the scene. Ambient lights support only the `color` light property described in Table 1.
+For Physically-Based Rendering (PBR) materials, this parameter is treated as having an environment map having this color value for image based lighting (IBL).
+
+#### Directional
+
+Directional lights are light sources that emit from infinitely far away in the direction of the -z axis. This light type uses the light properties `color` and `direction` described in Table 1.
+
+#### Point
+
+Point lights emit light in all directions over a distance. Point lights support the `color` property. `constantAttenuation`, `linearAttenuation` and `quadraticAttenuation` light properties embedded in the `positional` property describe the different attenuations.
+
+```javascript
+"extensions": {
+    "KHR_lights" : {
+        "lights": [
+            {
+                "positional": [

```

Comment:
typo: I guess "positional" is supposed to be a dictionary? should be curly brackets then not square...